linker/shared/src/main/scala/org/scalajs/linker/backend/javascript/Printers.scala

```diff
@@ -760,7 +770,7 @@ object Printers {
         print("]")
     }
 
-    protected def print(exportName: ExportName): Unit =
+    private def print(exportName: ExportName): Unit =
```

Comment:
Should this have ideally gone to the commit that changes the visibility of other members?